packages/web/app/src/lib/supertokens/third-party-email-password-react-oidc-provider.ts

```diff
@@ -30,7 +30,7 @@ export const getOIDCOverrides = (): UserInput['override'] => ({
           ? {
               ...input,
               options: {
-                preAPIHook: async options => {
+                preAPIHook: options => {
```

Comment:
now it's not a Promise :( and it should be.
The same applies to all the functions you changed from `async () => {}` to `() => {}`.

Reply:
yeah this PR needs more attention that I expected 😬, I'll close it for now